openedx/core/djangoapps/credit/api.py

```diff
@@ -263,6 +267,8 @@ def create_credit_request(course_key, provider_id, username):
 
     if created:
         credit_request.uuid = uuid.uuid4().hex
+    else:
+        credit_request.timestamp = datetime.datetime.now(pytz.UTC)
```

Comment:
I think `auto_now` updates when the model is saved.  In this case, I want to update the timestamp, add the timestamp to the "parameters" dictionary, then save both the new timestamp and the parameters in the model.  I was thinking that doing it this way would ensure that the parameters dictionary and the model `timestamp` field are consistent.